scripts/vscode.patch

```diff
@@ -864,6 +864,31 @@ index d8de423529..5953e920c3 100644
 +		Object.defineProperty(this._options, 'useSameOriginForRoot', {
 +			value: true,
 +		});
+diff --git a/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts b/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts
+index d4cd525812..6904db5518 100644
+--- a/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts
++++ b/src/vs/workbench/parts/welcome/page/electron-browser/vs_code_welcome_page.ts
+@@ -30,8 +29,0 @@ export default () => `
+-				<div class="section recent">
```

Comment:
I would prefer commenting this out `<!-- -->` so the patch can be as minimal and additive as possible, should help with potential merge conflicts when we update VS Code.

Reply:
Also, it might be a good idea for us to actually implement the open recent command at some point.